src/sfctl/helps/app.py

```diff
@@ -56,11 +56,11 @@
       parameters list. This would results in application using the default
       value of the parameters from the application manifest.
     parameters:
-        - name: --application-name
+        - name: --application-id
           type: string
-          short-summary: The name of the application
+          short-summary: The identity of the application.
           long-summary: "This is typically the full name of the application
-            with the 'fabric:' URI scheme. Starting from version 6.0,
+            without the 'fabric:' URI scheme. Starting from version 6.0,
```

Comment:
Can you double check this changed in the SDK? This was a bug last time.

Reply:
I changed this because the URL it was producing is incorrect.

Reply:
Can you test through the CLI, I found that the documentation was actually incorrect.

Reply:
I have tested through the CLI just now. App id works :)